python-spec/src/somacore/types.py

```diff
@@ -22,6 +31,16 @@ def is_nonstringy_sequence(it: object) -> TypeGuard[Sequence]:
     return not isinstance(it, (str, bytes)) and isinstance(it, Sequence)
 
 
+def to_string_tuple(obj: Union[str, Sequence[str]]) -> Tuple[str, ...]:
+    """Returns a tuple of string values.
+
+    If the input is a string, it is returned as a tuple with the sting as its
```

Comment:
<pre>
>>> to_string_tuple("one")
(<img src="https://github.com/user-attachments/assets/69f297d9-9af5-4cc6-afdc-1c000d13946b" width=61 height=92 alt="Sting, the musician">,)
</pre>

```suggestion
    If the input is a string, it is returned as a tuple with the string as its
```